src/gatsby/createPages/createDevelopmentApiPages.ts

```diff
@@ -0,0 +1,72 @@
+import { getDataOrPanic } from "./helpers";
+
+export default async ({ actions, graphql, reporter }) => {
+  const data = await getDataOrPanic(
+    `
+      query {
+        allOpenApi {
```

Comment:
nice! the cool thing is this makes it easy to build the sidebar or any other dynamic navigation